examples/Viewer.tsx

```diff
@@ -394,19 +393,21 @@ class Viewer extends React.Component<{}, ViewerState> {
                             <label htmlFor={id}>{id}</label>
                             <input
                                 type="checkbox"
-                                onClick={(event) =>
-                                    this.turnAgentsOnOff(event.target.value)
-                                }
+                                onClick={(event) => {
+                                    const target: { value: string; }  = event.target as HTMLInputElement;
+                                    return this.turnAgentsOnOff(target.value)
+                                }}
```

Comment:
target.value was giving a typescript error that I had gotten in the habit of ignoring since it had been there awhile